src/main/java/org/wildfly/security/ssl/SSLContextBuilder.java

```diff
@@ -275,24 +277,50 @@ public SSLContextBuilder setClientMode(final boolean clientMode) {
         final int sessionTimeout = this.sessionTimeout;
         final boolean wantClientAuth = this.wantClientAuth;
         final boolean needClientAuth = this.needClientAuth;
+        final boolean useCipherSuitesOrder = this.useCipherSuitesOrder;
+
         return new OneTimeSecurityFactory<SSLContext>(() -> {
             final SecurityFactory<SSLContext> sslContextFactory = SSLUtils.createSslContextFactory(protocolSelector, providerSupplier);
             // construct the original context
             final SSLContext sslContext = sslContextFactory.create();
             SSLSessionContext sessionContext = clientMode ? sslContext.getClientSessionContext() : sslContext.getServerSessionContext();
             sessionContext.setSessionCacheSize(sessionCacheSize);
             sessionContext.setSessionTimeout(sessionTimeout);
+            final X509KeyManager x509KeyManager = keyManagerSecurityFactory == null ? null : keyManagerSecurityFactory.create();
             final X509TrustManager x509TrustManager = trustManagerSecurityFactory.create();
             final boolean canAuthPeers = securityDomain != null && securityDomain.getEvidenceVerifySupport(X509PeerCertificateChainEvidence.class).mayBeSupported();
-            sslContext.init(keyManagerSecurityFactory == null ? null : new KeyManager[] {
-                keyManagerSecurityFactory.create()
+
+            if (ElytronMessages.tls.isTraceEnabled()) {
+                ElytronMessages.tls.trace("SSLContext initialization:" +
```

Comment:
OK for this one you can use tracef but you'll still want the ``isTraceEnabled()`` check because of varargs array creation.